ipatests/test_integration/test_ipa_cert_fix.py

```diff
@@ -278,52 +292,86 @@ def install(cls, mh):
             setup_dns=False, extra_args=['--no-ntp']
         )
 
-    def test_renew_expired_cert_replica(self):
+    @pytest.fixture
+    def move_date_check_replication(self):
+        # add user to check replication after renewal
+        testuser = 'testuser1'
+        password = 'Secret@123'
+
+        # move system date to expire certs
+        for host in self.master, self.replicas[0]:
+            move_date(host, 'stop', '+3years+1days')
+
+        yield
+
+        # check user on replica
+        stdin = (f"{self.master.config.admin_password}\n"
+                 "{self.master.config.admin_password}\n"
+                 "{self.master.config.admin_password}\n")
```

Comment:
You need to add the f before each line, otherwise only the first occurrence of admin_password will be replaced:
```
        stdin = (f"{self.master.config.admin_password}\n"
                 f"{self.master.config.admin_password}\n"
                 f"{self.master.config.admin_password}\n")
```